libsolidity/interface/CompilerStack.cpp

```diff
@@ -59,6 +59,8 @@ CompilerStack::CompilerStack(ReadFileCallback const& _readFile):
 void CompilerStack::setRemappings(vector<string> const& _remappings)
 {
 	vector<Remapping> remappings;
+	map<string, int> index;
```

Comment:
Maybe this can be renamed into `globalPrefixReferences`.  I guess the type is rather `map<string, size_t>`.